credentials/src/test/kotlin/web5/sdk/credentials/VerifiablePresentationTest.kt

```diff
@@ -161,19 +161,20 @@ class VerifiablePresentationTest {
     val keyManager = InMemoryKeyManager()
     val holderDid = DidKey.create(keyManager)
 
-    val header = JWSHeader.Builder(JWSAlgorithm.ES256K)
-      .keyID(holderDid.uri)
+    val header = JwsHeader.Builder()
+      .type("JWT")
+      .algorithm(Jwa.ES256K.name)
+      .keyId(holderDid.uri)
```

Comment:
noting that the key ID needs to be a did plus a key reference (e.g. `did:example:abcd#key-1`)

Reply:
oh yeah! fixed, thank you